data/sql/updates/pending_db_world/add-path-ids-ulduar-trash.sql

```diff
@@ -0,0 +1,8 @@
+--
+SET @GUID := 137481;
+DELETE FROM `creature_addon` WHERE `guid` IN (@GUID, @GUID+1, @GUID+2, @GUID+3);
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(@GUID, (@GUID)*10, 0, 0, 1, 0, 0, NULL),
+(@GUID+1, (@GUID+1)*10, 0, 0, 1, 0, 0, NULL),
+(@GUID+2, (@GUID+2)*10, 0, 0, 1, 0, 0, NULL),
+(@GUID+3, (@GUID+3)*10, 0, 0, 1, 0, 0, NULL);
```

Comment:
Missing aura 63610

Reply:
Oh yeah forgot I did those on browser on the old PR